app/views/admin/events/index.html.haml

```diff
@@ -4,7 +4,10 @@
       %h1
         Events
         = "(#{@events.length})" if @events.any?
-        .btn-group.pull-right
+
+        .pull-right
+          - if can? :create, Event
+            =link_to 'Add Event', new_admin_conference_program_event_path(@conference.short_title), :class => 'button btn btn-default btn-info'
```

Comment:
I got your point, it is just that we don't have an info button anywhere else. Yes, we should decide on a specific course of action for multiple buttons (perhaps not in this particular PR but in a new issue). Unfortunately, no, we don't have UI guidelines. In terms of guidelines (not UI), we got rubocop for styling and we use new ruby syntax for the code.